app/models/user.rb

```diff
@@ -81,10 +81,10 @@ def sync(data)
       end
     end
 
-    # Apparently NOT IN doesn't work if uuids is empty
-    uuids = data.map { |x| x[:uuid] } + [""]
+
+    uuids = data.map { |x| x[:uuid] }
```

Comment:
Checked with @czorek. This `uuid` should never be missing in any element of data